authentication/account/repository/identity_blackbox_test.go

```diff
@@ -43,6 +45,55 @@ func (s *IdentityRepositoryTestSuite) TestDelete() {
 		}
 	})
 
+	s.T().Run("ok hard delete by identity ID", func(t *testing.T) {
+		// given
+		g := s.NewTestGraph(t)
+		identity1 := g.CreateIdentity()
+		// create a second identity
+		g.CreateIdentity()
+
+		unscoped := func(s *gorm.DB) *gorm.DB {
+			return s.Unscoped()
+		}
+		// when hard delete user
+		err := s.Application.Identities().Delete(s.Ctx, identity1.ID(), unscoped)
+		// then
+		require.Nil(t, err)
+
+		identity, err := s.Application.Identities().Load(s.Ctx, identity1.ID(), unscoped)
+		require.EqualError(t, err, fmt.Sprintf("identity with id '%s' not found", identity1.ID()))
+		require.Nil(t, identity)
+
+		identities, err := s.Application.Identities().List(s.Ctx)
+		require.NoError(t, err, "Could not list identities")
+		require.True(t, len(identities) >= 1)
```

Comment:
Or you could use `require.NotEmpty(t, identities)`

Reply:
Addressed in 72b2576